gazebo_ros/launch/gzserver.launch.py

```diff
@@ -220,12 +232,16 @@ def _boolean_command(arg):
     return py_cmd
 
 
-# Add string commands if not empty
-def _arg_command(arg, condition=None):
-    if condition:
-        cmd = ['"--', arg, '" if "" != "', condition, '" else ""']
-    else:
-        cmd = ['"--', arg, '" if "" != "', LaunchConfiguration(arg), '" else ""']
+# Add string command and argument if not empty
+def _arg_command(arg, join_with=" "):
+    cmd = ['"--', arg, '" if "" != "', LaunchConfiguration(arg), '" else ""']
+    py_cmd = PythonExpression(cmd)
+    return (py_cmd, join_with, LaunchConfiguration(arg))
```

Comment:
It might be harmless but in the case that the launch configuration is an empty string you'll still be adding the join with. I think anyway.

Reply:
And actually if the join with is = that would be a problem no?

Reply:
I amended https://github.com/ros-simulation/gazebo_ros_pkgs/commit/eb1d266e70ba86fcb3f0aa7ed263d05d5650730b to https://github.com/ros-simulation/gazebo_ros_pkgs/pull/1456/commits/efa494f3285bd8516b1e32af925a72559e5232a7, and I think this fixes the handling of `join_with` when the argument is empty